Formula/glibc.rb

```diff
@@ -18,9 +18,15 @@ def message
 class GawkRequirement < Requirement
   fatal true
 
+  MINIMUM_GAWK_VERSION = "3.1.2".freeze
+
+  def gawk_version
+    @gawk_version ||= Version.new Utils.popen_read("gawk --version").split("\n").first.split.last
+  end
+
   satisfy(:build_env => false) do
     # Returning which("gawk") causes a cyclic dependency.
-    !which("gawk").nil?
+    which("gawk").present? and gawk_version >= MINIMUM_GAWK_VERSION
```

Comment:
```suggestion
    which("gawk").present?
```
Gawk 3.1.2 was released in 2003. I do not believe it is worth adding the code to verify that the version is sufficient.
https://ftp.gnu.org/gnu/gawk/

Reply:
what cutoff do you use to decide whether it should or shouldn't verify software version?

Reply:
The oldest version of `glibc` that we support is 2.13, and it was release on 2011-02-01, so let's say ~2010.
https://docs.brew.sh/Homebrew-on-Linux#requirements
https://sourceware.org/glibc/wiki/Glibc%20Timeline